platform/nodejs/index.js

```diff
@@ -25,6 +25,12 @@
 
 import { createRequire } from 'module';
 
+import { readFileSync } from 'fs';
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
```

Comment:
[`__dirname`](https://nodejs.org/docs/latest/api/modules.html#modules_dirname) is a build-in variable in Node.js, but it's not available in ES modules. This is the common workaround. We want to know the directory in which this file exists.